### PR TITLE
CSS-7421 add upgrades integration test

### PR DESCRIPTION
**Pull request by AmberCharitos**
Add integration test to validate that a charm deployed from the edge channel could be upgraded to the current charm without conflict, while maintaining configuration values.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -51,6 +51,20 @@ deps =
 commands =
     pytest {[vars]tst_path}integration/test_policy.py -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
 
+[testenv:integration-upgrades]
```
Comment (gtato) [inline tox.ini:54]: same comment as in the other PR

**Review by gtato — APPROVED** [review]
lg (with a minor comment)
